vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -136,6 +164,24 @@ public DateTimePicker(
         addValueChangeListener(listener);
     }
 
+    /**
+     * Convenience constructor to create a date time picker with a
+     * {@link ValueChangeListener} and a label.
+     *
+     *
+     * @param label
+     *            the label describing the date picker
```

Comment:
"date time picker" instead of "date picker"

Reply:
Updated.